hrm-domain/hrm-core/contact/contactService.ts

```diff
@@ -124,23 +125,35 @@ const permissionsBasedTransformations: PermissionsBasedTransformation[] = [
   },
 ];
 
+export const contactRecordToContact = (record: ContactRecord): Contact => {
+  const { id, caseId, ...rest } = record;
+  return {
+    ...rest,
+    id: record.id.toString(),
+    ...(record.caseId ? { caseId: record.caseId.toString() } : {}),
```

Comment:
Not asking this to change, but just to know: Is there a reason why `caseId: record.caseId?.toString()` does not works here?